dev/lib/product_taxonomy/models/mixins/localized.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ProductTaxonomy
+  module Localized
+    def localizations
```

Comment:
Should this be private? I suppose not strictly required, but I usually try to minimize method polution when metaprogramming

Reply:
True, will make them private